src/components/WeeklySummary/WeeklySummary.css

```diff
@@ -110,6 +110,11 @@
   border-color: #fff #dee2e6 #dee2e6;
 }
 
+.summary-instructions-row {
+  display: flex;
+  justify-content: space-between;
```

Comment:
```suggestion
  justify-content: space-between;
  margin-right: 0;
```
Last thing so the button lines up with the editor, but everything looks good other than that.
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/39d6713d-a034-4749-bf86-08449eb45c1f)